inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/mvn.cpp

```diff
@@ -38,13 +38,19 @@ const std::vector<double> epsilon = {
     0.000000001
 };
 
+const std::vector<std::map<std::string, std::string>> Configs = {
+        {},
+        {{CONFIG_KEY(DYN_BATCH_ENABLED), CONFIG_VALUE(YES)}}
+};
+
```

Comment:
To check dynamic batch you should pass real batch size into infer request via call of method `InferRequest::SetBatch(...)`. Not only specify config key. Please improve test body to check this functionality correctly.